spec.html

```diff
@@ -1760,17 +1768,17 @@ <h1>The Number Type</h1>
         <div class="math-display">
           _s_ &times; _m_ &times; 2<sup>_e_</sup>
         </div>
-        <p>where _s_ is 1 or -1, _m_ is an integer such that 2<sup>52</sup> &le; _m_ &lt; 2<sup>53</sup>, and _e_ is an integer such that -1074 &le; _e_ &le; 971.</p>
+        <p>where _s_ is 1 or -1, _m_ is an integer in the interval from 2<sup>52</sup> (inclusive) to 2<sup>53</sup> (exclusive), and _e_ is an integer in the inclusive interval from -1074 to 971.</p>
```

Comment:
Fairly neutral on using intervals here, what do you think is gained from the current way @michaelficarra?

Reply:
I'm pretty neutral on this one, too, but I would err on the side of consistent phrasing. That means using intervals wherever it doesn't detract from readability.

Reply:
Fair enough.